File: c/functions/3-factorial.c
#include"stdio.h"
void factorial();
void main(){
	factorial();
}
void factorial(){
	int i, n, a=1;
	printf("Enter a number to get its factorial: ");
	scanf("%d", &n);
	for (i=1; i<=n; i++){
		a*=i;
	}
	printf("%d\n", a);
}

File: java/9-day/3-factorial.java
import java.util.Scanner;

public class factorial {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int num = scan.nextInt();
        int factorial = 1;

        for (int i = 1; i <= num; i++) {
            factorial *= i;
        }
        System.out.println(
            "The factorial of number " + num + " is " + factorial
        );
        scan.close();
    }
}
